client/src/lib/constants.ts

```diff
@@ -2,9 +2,12 @@ import { Commitment, PublicKey } from '@solana/web3.js';
 import { ClusterType } from '@identity.com/sol-did-client';
 
 // Should equal the contents of program/program-id.md
-export const PROGRAM_ID: PublicKey = new PublicKey(
-  'boxndjnzQZEWbBku3YipL4pchYRc1zi4nNSrFUwawWo'
-);
+const PROD_PROGRAM_ID = 'boxndjnzQZEWbBku3YipL4pchYRc1zi4nNSrFUwawWo';
+const DEV_PROGRAM_ID = 'dbo6b6DDmWTWpLcZM5SoVEf9X8DSJYPjVfD3Ptz6hod';
+
+export const STAGE = (process.env.REACT_APP_STAGE || process.env.NODE_ENV);
+
+export const PROGRAM_ID: PublicKey = STAGE === 'development' ? new PublicKey(DEV_PROGRAM_ID) : new PublicKey(PROD_PROGRAM_ID);
```

Comment:
Maybe we can generalize this to easily support other configurations in the future? Like an overall envConfig object with keys `deveopment`, `production` and the configuration could have a generator function the publicKey: `envConfig['development'].createPublicKey()`.

Reply:
Yeah that sounds good.